aemanalyser-maven-plugin/pom.xml

```diff
@@ -101,27 +93,34 @@ governing permissions and limitations under the License.
   </build>
 
   <dependencies>
+    <!-- all dependencies in scope "provided" are exported by the Maven Core Classloader
+         and their version is given through the Maven distribution -->
     <dependency>
       <groupId>org.apache.maven</groupId>
       <artifactId>maven-core</artifactId>
       <version>${maven.version}</version>
+      <scope>provided</scope>
     </dependency>
     <dependency>
       <groupId>org.apache.maven</groupId>
       <artifactId>maven-plugin-api</artifactId>
       <version>${maven.version}</version>
-    </dependency>
-    <dependency>
-      <groupId>org.apache.maven.plugin-tools</groupId>
-      <artifactId>maven-plugin-annotations</artifactId>
-      <version>${maven.version}</version>
+      <scope>provided</scope>
     </dependency>
     <dependency>
       <groupId>org.apache.maven</groupId>
       <artifactId>maven-artifact</artifactId>
       <version>${maven.version}</version>
+      <scope>provided</scope>
+    </dependency>
+
+    <dependency>
+      <groupId>org.apache.maven.plugin-tools</groupId>
+      <artifactId>maven-plugin-annotations</artifactId>
+      <version>3.6.1</version><!-- same version as m-plugin-p -->
+      <scope>provided</scope><!-- no run time retention -->
```

Comment:
Plugin version from parent pom is 3.6.0 - maybe we should move the plugin definition from the parent pom into this one?

Reply:
Would be good to manage both versions with the same property as version value in parent.

Reply:
yes, either parent or this pom

Reply:
Done in https://github.com/adobe/aemanalyser-maven-plugin/pull/126/commits/de6a6ade9c7daa0961812731caebee97111bc2e5